Cargo.toml

```diff
@@ -31,10 +31,10 @@ features = ["nightly", "simd_backend"]
 travis-ci = { repository = "dalek-cryptography/curve25519-dalek", branch = "master"}
 
 [dev-dependencies]
-sha2 = { version = "0.9", default-features = false }
-bincode = "1"
-criterion = "0.3.0"
-hex = "0.4.2"
+sha2 = { version = "0.10", default-features = false }
+bincode = "1.3.3"
+criterion = "0.3.5"
+hex = "0.4.3"
```

Comment:
These are all semver equivalent to my knowledge.

Reply:
Oh sorry, not the sha2 change, I don't know why that got included in the diff, just the bincode, criterion, and hex changes.

Reply:
I was thinking I'd update the MCF forks in line with what we've been doing, then send another PR merging into upstraem dalek-crypto:master